inc/ec_util.h

```diff
@@ -519,20 +518,20 @@ class ec_util {
 	 * This function unwraps a wrapped attribute using the provided key and
 	 * optional additional authenticated data (AAD) from the frame.
 	 *
-	 * @param[in] wrapped_attrib The wrapped attribute to unwrap (retrieved using `get_attribute`).
+	 * @param[in] wrapped_attrib The wrapped attribute to unwrap (retrieved using `get_attrib`).
 	 * @param[in] frame The frame to use as AAD. Can be NULL if no AAD is needed.
 	 * @param[in] uses_aad Whether the wrapped attribute uses AAD.
 	 * @param[in] key The key to use for decryption.
 	 *
 	 * @return std::pair<uint8_t*, uint16_t> A heap allocated buffer of unwrapped attributes on success,
-	 *         which can then be fetched via `get_attribute`, along with the length of that buffer.
+	 *         which can then be fetched via `get_attrib`, along with the length of that buffer.
 	 *         The buffer is NULL and the size is 0 on failure.
 	 *
 	 * @warning The caller is responsible for freeing the memory returned by this function.
 	 *
-	 * @note Forwards to `unwrap_wrapped_attrib(ec_attribute_t *wrapped_attrib, uint8_t *frame, size_t frame_len, uint8_t *frame_attribs, bool uses_aad, uint8_t *key);`
+	 * @note Forwards to `unwrap_wrapped_attrib(ec_attribute_t wrapped_attrib, uint8_t *frame, size_t frame_len, uint8_t *frame_attribs, bool uses_aad, uint8_t *key);`
 	 */
-	static std::pair<uint8_t*, uint16_t> unwrap_wrapped_attrib(ec_attribute_t* wrapped_attrib, ec_frame_t *frame, bool uses_aad, uint8_t* key);
+	static std::pair<uint8_t*, uint16_t> unwrap_wrapped_attrib(ec_attribute_t wrapped_attrib, ec_frame_t *frame, bool uses_aad, uint8_t* key);
```

Comment:
Performance improvement, can you pass by `const` reference here since we are no longer using pointers.
```suggestion
	static std::pair<uint8_t*, uint16_t> unwrap_wrapped_attrib(const ec_attribute_t& wrapped_attrib, ec_frame_t *frame, bool uses_aad, uint8_t* key);
```